src/lumigo_tracer/spans_container.py

```diff
@@ -266,7 +272,7 @@ def create_span(cls, event=None, context=None, force=False) -> "SpansContainer":
         additional_info = {}
         if Configuration.verbose:
             additional_info.update(
-                {"event": EventDumper.dump_event(event), "envs": lumigo_dumps(dict(os.environ))}
```

Comment:
I think that it better to just pass `event` to SpansContainer, and then the `envs` and `event` could be calculated inside the instance 🤔

Reply:
I don't understand what do you mean. Do you mean that it should be a function inside the `SpansContainer` that updates `event` and `envs`?
In this part of the code, `SpansContainer` still doesn't exist.

Reply:
I know, it will be initialized a few lines later.
I meant to pass the event to the SpansContainer as a parameter, and then you can avoid the parsing in `tracer.py`